### DISCUSSION TITLE
Having issue with cables bulk import

### DISCUSSION THREAD

**Discussion by CBARHBS** [netbox-community/netbox#8890] [Q&A]
Hi,

I am having the below issue when I try to bulk import some cables in Netbox v3.1.9
![image](https://user-images.githubusercontent.com/86604976/158689156-f0cc6cda-1676-4382-8c24-617afdaf043c.png)

Importing the same data works without any problem in Netbox v3.1.8

I don't have another instance of Netbox 3.1.9 therefore I am not sure if the problem is reproducible.

Did anyone reported this kind of issue before ?

Regards

Christophe

**Comment by drmsoffall** [comment]
If you scroll down past the input box for Cable Bulk Import, there's a list of available fields.  It does not appear that `termination_a_id` is a valid field anymore.  You will probably need to refactor your data to successfully import it with v3.1.9.

**Reply by rogerdat101** [reply]
Seeing this error in v3.1.11.  Correct, there isn't a valid field for termination_a_id or b so what is the solution here?  Not sure what "refactor your data" is supposed to mean with regards to the last response in this thread?

**Reply by candlerb** [reply]
It means "change your data to match the required format".  If you go to `/dcim/cables/import/` you'll get a table below the data import field, showing all the permitted columns.

Can you give an example of the CSV you're trying to import? (Heading row plus one data row)  It seems very odd that this would have changed between 3.1.8 and 3.1.9 though.
